PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -27,6 +27,13 @@ def miniAOD_customizeCommon(process):
 
     process.patMuons.computeMiniIso = cms.bool(True)
     
+    # EGamma objects from HGCal are not yet in GED
+    # so add companion collections for Phase-II MiniAOD production
+    from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+    from PhysicsTools.PatAlgos.slimming.addHGCalEgammaCollections import addHGCalEgammaCollections
+    if phase2_hgcal.isChosen():
```

Comment:
@kpedro88 
I recall that you mentioned (in ORP?) that 93X updates to miniAOD will come without modifying the default miniAOD of 93X.
This PR changes the default.
Please clarify.

Reply:
@slava77 though the use of eras can be improved (as commented in the master branch PR), this doesn't change the default 93X miniAOD.